pkgs/tools/misc/yt-dlp/default.nix

```diff
@@ -10,18 +9,20 @@
 
 buildPythonPackage rec {
   pname = "yt-dlp";
-  # The websites yt-dlp deals with are a very moving target. That means that
-  # downloads break constantly. Because of that, updates should always be backported
-  # to the latest stable release.
-  version = "2021.08.02";
-
-  src = fetchFromGitHub {
-    owner = "yt-dlp";
-    repo = "yt-dlp";
-    rev = version;
-    sha256 = "QEJKOZGVQNXLU8GfTbwBx2Zv3KO++ozTJcXLWxXA4hI=";
+  version = "2021.08.10";
+
+  src = fetchPypi {
+    inherit pname;
+    version = with lib; concatMapStringsSep "." (removePrefix "0") (with versions; [(major version) (minor version) (patch version)]);
```

Comment:
I don't know a better way to remove the leading zeros from package version, suggestions welcome.

Reply:
We cna also just change the version number.

Reply:
The one without leading zeros seems to be only used for pypi. This just looks not very nice but shouldn't be a real trouble.

youtube-dl is in the same fashion (program version is `2021.06.06` while pypi is `2021.6.6`), however their version is "normalized" by some certain script (I didn't get to remember which one it is, may try again later).

Reply:
Another way would be to do

```nix
builtins.replaceStrings [ ".0" ] [ "." ] "2020.01.01"
```

I think that should be slightly faster to compute